packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx

```diff
@@ -251,7 +251,7 @@ export const IntercomCard: React.FC<
       );
       break;
     case 'loadingView':
-      view = null;
+      view = <div>loading state</div>;
```

Comment:
Is this an intermediary step? Not sure if we want to show 'loading state'

Reply:
Yeah, let's make this user-friendly:
`<div>Loading&#8230;</div>`

Maybe better to have a separate PR for this just in case

Reply:
loading state is for testing, forgot to remove it. Updated with <div>Loading&#8230;</div>